exporter/awsemfexporter/datapoint.go

```diff
@@ -156,6 +164,79 @@ func (dps histogramDataPointSlice) CalculateDeltaDatapoints(i int, instrumentati
 	}}, true
 }
 
+// CalculateDeltaDatapoints retrieves the ExponentialHistogramDataPoint at the given index.
+func (dps exponentialHistogramDataPointSlice) CalculateDeltaDatapoints(i int, instrumentationScopeName string, detailedMetrics bool) ([]dataPoint, bool) {
+	metric := dps.ExponentialHistogramDataPointSlice.At(i)
+	labels := createLabels(metric.Attributes(), instrumentationScopeName)
+	timestamp := unixNanoToMilliseconds(metric.Timestamp())
+
+	scale := metric.Scale()
+	base := math.Pow(2, math.Pow(2, float64(-scale)))
+	datapoints := []dataPoint{}
+	arrayValues := []float64{}
+	arrayCountes := []float64{}
+
+	// Set mid-point of positive buckets in values/counts array.
+	positiveBuckets := metric.Positive()
+	positiveOffset := positiveBuckets.Offset()
+	positiveBucketCounts := positiveBuckets.BucketCounts()
+	for i := 0; i < positiveBucketCounts.Len(); i++ {
+		index := i + int(positiveOffset)
+		bucketBegin := math.Pow(base, float64(index))
+		bucketEnd := math.Pow(base, float64(index+1))
+		metricVal := (bucketBegin + bucketEnd) / 2
```

Comment:
`bucketBegin` [isn't actually in the bucket](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#exponential-buckets).

```suggestion
		metricVal := ((bucketBegin + 1) + bucketEnd) / 2
```

Reply:
It is left-open right-closed range like (bucketBegin, bucketEnd] for positive bucket. The mid-point of bucket is calculated as  (bucketBegin + bucketEnd) / 2. An example is as below.
When scale=-1 and offset=-1, base = 2**(2**(-scale)) = 4.
1. The first bucket index = offset + 0 = -1. The first bucket range is (0.25, 1]. The mid-point is (0.25+1)/2 = 0.625
2. The second bucket index = offset + 1 = 0. The second bucket range is (1, 4]. The mid-point is (1+4)/2 = 2.5
3. The third bucket index = offset + 2 = 1. The third bucket range is (4, 16]. The mid-point is (4+16)/2 = 10